boot/configure.ml

```diff
@@ -58,7 +59,8 @@ let () =
     v := Some dir
   in
   let args =
-    [ ( "--libdir"
+    [ "--prefix", Arg.String (set_dir prefix), "DIR where files are copied"
```

Comment:
is there logic missing from this PR?

I se you `set_dir prefix` but then `prefix` doesn't get used anywhere?